Logger/FileLogger.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public string FileName { get; private set; }
```

Comment:
Auto property can be made readonly.

```suggestion
        public string FileName { get; }
```